package.json

```diff
@@ -11,36 +11,36 @@
 		"can-release": "if [ \"$TRAVIS_EVENT_TYPE\" = cron ] && [ $(git rev-list -n 1 --since=\"26 hours ago\" master) ]; then true; else false; fi"
 	},
 	"dependencies": {
-		"copy-text-to-clipboard": "^1.0.2",
+		"copy-text-to-clipboard": "^1.0.3",
 		"debounce-fn": "^1.0.0",
 		"dom-chef": "^1.3.0",
 		"dom-loaded": "^1.0.0",
-		"element-ready": "^2.0.0",
-		"github-injection": "^0.3.0",
+		"element-ready": "^2.2.0",
+		"github-injection": "^1.0.1",
 		"linkify-issues": "^1.3.0",
 		"linkify-urls": "^1.3.0",
 		"select-dom": "^4.1.0",
 		"shorten-repo-url": "^1.1.0",
 		"to-markdown": "^3.1.0",
 		"to-semver": "^1.1.0",
 		"webext-dynamic-content-scripts": "^2.0.1",
-		"webext-options-sync": "^0.11.0",
-		"webextension-polyfill": "^0.1.1"
+		"webext-options-sync": "^0.12.0",
+		"webextension-polyfill": "^0.2.1"
 	},
 	"devDependencies": {
-		"ava": "*",
```

Comment:
Was this a purposeful change to remove the `*`?

Reply:
Nope, fixing